pages/runtime/data-api.md

```diff
@@ -0,0 +1,676 @@
+# Data API Service
+
+## Introduction
+Resin's Data API Service constitutes a central node in resin architecture, since it provides interconnection between resin services and communication with the database. At the same time, it exposes an HTTP interface, based on Open Data ([OData](http://www.odata.org)) format, that provides a means to the user to interact with resin resources and fetch or update information regarding their applications, devices and environment variables.
+
+This document aims to describe the endpoints that are provided by the Data API, as well as showing examples of their expected usage.
+
+The requests are expected to follow the OData format, and the responses are provided in JSON format. The authorization token that is required in each request can be found under 'Preferences' in the user's dashboard.
+
+## Interacting with resources
+
+### Resource: Application
+#### Get all applications
+* Summary: Get all applications along with the corresponding information (application name, id, device type, git repository, latest commit)
+* Endpoint: `https://api.resin.io/v1/application`
```

Comment:
Completely up to you -- I'd suggest you documented the base URL and API version in the intro and then just specify the endpoint, i.e. ``/application``